db/view/view_update_generator.cc

```diff
@@ -63,6 +64,8 @@ future<> view_update_generator::start() {
                             ::streamed_mutation::forwarding::no,
                             ::mutation_reader::forwarding::no);
 
+                    utils::get_local_injector().inject("view_update_generator_consume_staging_sstable",
+                            [] { throw std::runtime_error("consuming staging sstable failed"); });
```

Comment:
I prefer how in the next patch you added a function that replaces these two lines, which are long and ugly and force you to repeat the injected failure name twice.

Reply:
Sure, I can do that in this file too.